src/test/java/streams/part1/exercise/Exercise1.java

```diff
@@ -17,7 +19,9 @@ public void findPersonsEverWorkedInEpam() {
         List<Employee> employees = Example1.getEmployees();
 
         // TODO реализация, использовать Collectors.toList()
-        List<Person> personsEverWorkedInEpam = null;
+        List<Person> personsEverWorkedInEpam = employees.stream().filter(e -> e.getJobHistory().stream()
+                                                                                                .anyMatch(jobHistoryEntry -> "EPAM".equals(jobHistoryEntry.getEmployer())))
+                                                                .map(Employee::getPerson).collect(Collectors.toList());
```

Comment:
No changes needed, but please, in further tasks, can u place each stream method called in chain one under another, just like Nikolai showed?
It really makes the code more readable.